Bottom: 37a368b15944f1a1662d6035abecbd9009b6fd5e
Top:    e22327339c17a84aca21d2dfe7588fa7aea241dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 19:18:26 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
